code/modules/jobs/job_types/bartender.dm

```diff
@@ -18,6 +18,13 @@
 
 	family_heirlooms = list(/obj/item/reagent_containers/glass/rag, /obj/item/clothing/head/that, /obj/item/reagent_containers/food/drinks/shaker)
 
+	mail_goodies = list(
+		/obj/item/storage/box/rubbershot = 30,
+		/datum/reagent/consumable/clownstears = 10,
+		/obj/item/stack/sheet/mineral/plasma{amount = 2} = 10,
```

Comment:
I'd prefer if you didn't make anonymous subtypes, and instead just make a new `plasma/two` subtype.

Reply:
having a subtype for all of the different amounts of mineral sheets would be confusing imo, especially if you want to place one that doesnt conform and dont want to have plasma/two, plasma/three, plasma/four, plasma/twentyseven in the future